Library/Tests/ViewModels/LiveStreamContainerViewModelTests.swift

```diff
@@ -173,7 +173,7 @@ internal final class LiveStreamContainerViewModelTests: TestCase {
 
     withEnvironment(liveStreamService: MockLiveStreamService(fetchEventResult: Result(nonLiveStreamEvent))) {
       self.vm.inputs.configureWith(project: project,
-                                   liveStreamEvent: nonLiveStreamEvent,
+                                   liveStreamEvent: liveStreamEvent,
```

Comment:
This test was actually not correct, we wanted to first configure with a live event and poll until we receive another live event from the api. So this change picked that up!